dd-java-agent/instrumentation/vertx/src/rxHandlerTests/groovy/VertxRxServerTest.groovy

```diff
@@ -0,0 +1,119 @@
+// Modified by SignalFx
+
+
+import datadog.trace.agent.test.AgentTestRunner
+import datadog.trace.agent.test.utils.OkHttpUtils
+import datadog.trace.agent.test.utils.PortUtils
+import io.vertx.reactivex.core.Vertx
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import spock.lang.Shared
+
+import static datadog.trace.agent.test.base.HttpServerTest.ServerEndpoint.SUCCESS
+
+class VertxRxServerTest extends AgentTestRunner {
+  static {
+    System.setProperty("dd.integration.jdbc.enabled", "true")
```

Comment:
Not jumping out to me why this is necessary if the instrumentation is included with the test task.

Reply:
I don't know all details how this flag is used, but it is even necessary in https://github.com/signalfx/signalfx-java-tracing/blob/3f8a80189bf942b9ac32c419cd0741cd8ef5e8cf/dd-java-agent/instrumentation/jdbc/src/test/groovy/JDBCInstrumentationTest.groovy#L30